File: AtlasWeb/Core/Modules/Tibbi_Surec/Saglik_Kurulu_Modulu/LowerExtremityFormViewModel.ts
//$90F21BC1
import { LowerExtremity } from "NebulaClient/Model/AtlasClientModel";
import { BaseHCExaminationDynamicObjectFormViewModel } from "Modules/Tibbi_Surec/Saglik_Kurulu_Modulu/BaseHCExaminationDynamicObjectFormViewModel";

export class LowerExtremityFormViewModel extends BaseHCExaminationDynamicObjectFormViewModel {
    protected __type__: string = 'Core.Models.LowerExtremityFormViewModel, Core';
    public _LowerExtremity: LowerExtremity = new LowerExtremity();
    public IsReadOnly: boolean;
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Saglik_Kurulu_Modulu/LowerExtremityFormViewModel.cs
//$565AECFD
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class LowerExtremityServiceController
    {
    }
}

namespace Core.Models
{
    public partial class LowerExtremityFormViewModel: BaseHCExaminationDynamicObjectFormViewModel
    {

        public override void AddBaseHCExaminationDynamicObjectFormViewModelToContext(TTObjectContext objectContext)
        {
            LowerExtremityServiceController lowerExtremityServiceController = new LowerExtremityServiceController();
            lowerExtremityServiceController.LowerExtremityFormInternal(this,  objectContext);
        }

        public bool IsReadOnly { get; set; }
    }
}
